`make install` is failing!

After the just released 1.5.5, executing `make install` is giving this error:

```
/Users/david.roman/Development/swift/Sources/Danger/DangerDSL.swift:19:2: error: 'mutating' may only be used on 'func' declarations
 mutating
 ^~~~~~~~
```

It's failing to build on CI for me as well, we should try to hotfix asap.

--- a/Sources/Danger/DangerDSL.swift
+++ b/Sources/Danger/DangerDSL.swift
@@ -16,7 +16,6 @@ public struct DangerDSL: Decodable {
     public private(set) var github: GitHub!
 
     public let bitbucketServer: BitBucketServer!
- mutating
     
     public let utils: DangerUtils
 

